packages/vite/src/node/utils.ts

```diff
@@ -733,3 +733,9 @@ export function parseRequest(id: string): Record<string, string> | null {
 }
 
 export const blankReplacer = (match: string) => ' '.repeat(match.length)
+
+export const stringsRE = /"[^"]*"|'[^']*'|`[^`]*`/g
```

Comment:
should this regex handle escaped closing quotes  `"like \" this"` or multiline
```
`like
this`
```

Reply:
Oh, this isn't part of the PR. I did a bad merge here.